redirects-transition.conf

```diff
@@ -1,9 +1,8 @@
 #Redirect rules for specific pages
 
-rewrite ^/eeo/documents/annual_no_fear_act.pdf https://www.fec.gov/resources/cms-content/documents/annual_no_fear_act.pdf redirect;
-rewrite ^/info/guidance/debt_retirement.shtml https://www.fec.gov/help-candidates-and-committees/handling-loans-debts-and-advances/ redirect;
+
+
 rewrite ^/info/ElectionDate/ https://www.fec.gov/help-candidates-and-committees/dates-and-deadlines/ redirect;
-rewrite ^/info/publications/fecglossary2009.pdf https://www.fec.gov/help-candidates-and-committees/guides/ redirect;
```

Comment:
I don't think you meant to remove the transition redirects right? You just wanted to update this PR with those redirects you put in Wagtail? https://github.com/fecgov/fec-proxy/pull/128/files